src/NuGet.Core/NuGet.Commands/RestoreCommand/RestoreRunner.cs

```diff
@@ -259,7 +259,7 @@ public static async Task<RestoreSummary> CommitAsync(RestoreResultPair restoreRe
             var log = summaryRequest.Request.Log;
 
             // Commit the result
-            log.LogInformation(Strings.Log_Committing);
+            log.LogVerbose(Strings.Log_Committing);
```

Comment:
I seriously considered removing this. 

It literally just says "Comitting restore" which is not extremely helpful but it is a progress indicator. 

For now, I just moved it verbose, but we can totally change it if others strongly lean one way or another.